Exercises/strings/strings.js

```diff
@@ -0,0 +1,18 @@
+// STRINGS
+// 1. Assign string with unicode to a variable.
+let unicodeStr = 'Example of unicode symbol is \u00B9';
+console.log(unicodeStr);
+
+// 2. Declare a variable with string with escaped symbols.
+let escapeSymb = '\u00A5';
+console.log(escapeSymb);
```

Comment:
it's better not to cut words in variable names

Reply:
Resolved.